src/m365/spo/commands/site/site-set.ts

```diff
@@ -201,6 +211,20 @@ class SpoSiteSetCommand extends SpoCommand {
     });
   }
 
+  private setGroupifiedSiteDescription(logger: Logger, args: CommandArgs): Promise<void> {
```

Comment:
Since we're not using the `logger`, there is no need to include it as the function's argument